views/firstLogin/sections/welcome_newpw.hbs

```diff
@@ -0,0 +1,8 @@
+<section data-panel="section-{{section-nr}}" {{#if submit-page}}class="submit-page" {{/if}}>
+  <h2 class="h4">Herzlich willkommen in der {{@root.theme.short_title}}, {{@root.currentUser.firstName}} {{@root.currentUser.lastName}}</h2>
+  <p>Wir freuen uns, dass du die {{@root.theme.short_title}} nutzen möchtest.</p>
+  <p>Ein Administrator hat für deinen Account ein neues Passwort vergeben. Bitte lege ein neues Passwort fest, dann kann es auch schon weitergehen. 😊</p>
+  
+  {{> "firstLogin/partials/language-hint"}}
+  {{#if @root.sso}}<input type="hidden" name="sso" value="{{@root.sso}}" />{{/if}}
```

Comment:
Is the SSO case even possible here? 

And where can I find the input fields to change my password? If there is no input, I think the text is missleading.